roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -1,107 +1,453 @@
+-   `Seconded` statements may only be sent to other members of the group when the
+    candidate is fully known by the validator.
+-   Validators in a cluster receiving messages about unknown candidates request
+    the candidate (and statements) from other cluster members which have it.
+-   Spam considerations
+    -   The maximum depth of candidates allowed in asynchronous backing determines the
+        maximum amount of `Seconded` statements originating from a validator V which
+        each validator in a cluster may send to others. This bounds the number of
+        candidates.
+    -   There is a small number of validators in each group, which further limits
+        the amount of candidates.
+
+&#x2014;
+
+-   We accept candidates which don't fit in the fragment trees of any relay
+    parents.
+    -   We listen to prospective parachains subsystem to learn of new additions to
+        the fragment trees.
+    -   Use this to attempt to import the candidate later.
+
+### Grid Mode
+
+-   Every consensus session provides randomness and a fixed validator set, which
+    is used to build a redundant grid topology.
+-   This grid topology is used to create "sending" and "receiving" paths from each
+    validator group to every validator.
+-   When a node observes a candidate as backed, it sends a
+    `BackedCandidateManifest` to their "receiving" nodes.
+-   If receiving nodes don't yet know the candidate, they request it.
+-   Once they know the candidate, they respond with a
+    `BackedCandidateAcknowledgement`.
+-   Once two nodes perform a manifest/acknowledgement exchange, they can send
+    `Statement` messages directly to each other for any new statements they might
+    need.
+-   There are limitations on the number of candidates that can be advertised by
+    each peer, similar to those in the cluster. Validators do not request
+    candidates which exceed these limitations.
+-   Validators request candidates as soon as they are advertised, but do not
+    import the statements until the candidate is part of the hypothetical
+    frontier, and do not re-advertise or acknowledge until the candidate is
+    considered both backed and part of the hypothetical frontier.
+-   Note that requesting is not an implicit acknowledgement, and an explicit
+    acknowledgement must be sent upon receipt.
+
+### Old Grid Protocol
+
+-   Once the candidate is backed, wait for some time T to pass to collect
+    additional statements. Then produce a 'backed candidate packet'
+    `(CommittedCandidateReceipt, Statements)`.
+-   Members of a backing group produce an announcement of a fully-backed candidate
+    (aka "full manifest") when they are finished.
+    -   `BackedCandidateManifest`
+    -   Manifests are sent along the grid topology to peers who have the relay-parent
+        in their implicit view.
+    -   Only sent by 1st-hop nodes after downloading the backed candidate packet
+        -   TODO: why?
```

Comment:
They're only re-sent by 1st-hop nodes because the grid topology is a 2-dimensional grid that provides either a 1 or 2-hop path from any originator to any recipient - 1st-hop nodes are those which share either a row or column with the originator, and 2nd-hop nodes are those which share a column or row with that 1st-hop node.

Note that for the purposes of statement distribution, we actually take the union of the routing paths from each validator in a group to the local node to determine the sending and receiving paths.